docs/website/binding_types_reference_guide.md

```diff
@@ -720,6 +720,28 @@ class.
 
  <a name="NoDefaultValueAttribute"></a>
```

Comment:
seems this is inserted in the wrong place

Reply:
Whoops, yeah nice catch!

Reply:
@spouliot fixed, thanks!